package.json

```diff
@@ -2,24 +2,26 @@
   "name": "fdk-harvest-admin",
   "version": "1.0.0",
   "description": "",
-  "main": "src/app.ts",
   "scripts": {
+    "dbup": "docker-compose -f docker-compose-mongo.yml up -d mongodb ",
+    "dbdown": "docker-compose -f docker-compose-mongo.yml stop mongodb ",
     "dev": "NODE_ENV=development ts-node-dev --respawn --no-notify --transpileOnly ./src/server.ts",
     "build": "rm -rf dist && tsc",
-    "lint": "tsc --noEmit && eslint ./ --ext=js,jsx,ts,tsx --fix",
+    "prod": "NODE_ENV=production node ./dist/src/server.js",
+    "lint": "tsc --noEmit && eslint ./src ./test --ext=js,ts --fix",
     "test": "NODE_ENV=test node node_modules/mocha/bin/_mocha -r ts-node/register test/data-source.api.test.ts --exit --timeout 10000"
   },
   "type": "module",
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "app-root-path": "^2.2.1",
```

Comment:
Can we get by without this dependency?

Reply:
We can if there is a native way of resolving app root directory. Though it seems like it's very common to use this dependency.

Reply:
tsc flytter kode i en dist,  og derfor endrer path. spec står på root